src/openloco/openloco.cpp

```diff
@@ -698,7 +699,8 @@ namespace openloco
         audio::update_ambient_noise();
         call(0x00444387);
 
-        addr<0x009C871C, uint8_t>() = addr<0x00F25374, uint8_t>();
+        // TODO: does this apply to the original?
+        s5::configuration().madeAnyChanges = addr<0x00F25374, uint8_t>();
```

Comment:
@AaronVanGeffen @IntelOrca is the change applied to the original data here? Or does it create a local copy of the struct and then discards it.

Reply:
You're letting `s5::configuration()` return a reference, so this should be modifying the original data, yes.